### PR TITLE
Corrected the Authentication Request Matcher for Authentication Filte…

### PR DESCRIPTION
**Pull request by btundes**
Please review and Merge  #1110

### CODE REVIEW

File: source/Security/Authentication.rst [hunk 1]
```diff
@@ -1708,7 +1708,7 @@ Spring Securityから提供されている\ `認証プロバイダ <http://docs.
         class="com.example.app.common.security.CompanyIdUsernamePasswordAuthenticationFilter">
         <!-- (5) -->
         <property name="requiresAuthenticationRequestMatcher">
-            <bean class="org.springframework.security.web.authentication.logout.LogoutFilter$FilterProcessUrlRequestMatcher">
+            <bean class="org.springframework.security.web.util.matcher.AntPathRequestMatcher">
                 <constructor-arg value="/authentication" />
             </bean>
```
Comment (btshimizukza) [inline source/Security/Authentication.rst]: Please specify a "POST" in second constructor argument (HTTP method of processing target).

for example : 

```xml
<bean class="org.springframework.security.web.util.matcher.AntPathRequestMatcher">
     <constructor-arg index="0" value="/authentication" />
     <constructor-arg index="1" value="POST" /> <!-- ### Specify this argument ### -->
</bean>
```

An above definition is equivalent to when use a Java Configuration mechanism.
For details, refer to [here](https://github.com/spring-projects/spring-security/blob/3.2.x/config/src/main/java/org/springframework/security/config/annotation/web/configurers/FormLoginConfigurer.java#L224-L227).

And in the Spring Security 4(latest version), xml namespace parser specify a "POST" method. (refer to [here](https://github.com/spring-projects/spring-security/blob/master/config/src/main/java/org/springframework/security/config/http/FormLoginBeanDefinitionParser.java#L165-L170))

**Comment by btshimizukza** [comment]
@btundes 

Please confirm a [review comment](https://github.com/terasolunaorg/guideline/pull/1111#discussion_r31982788).

**Comment by btundes** [comment]
@btshimizukza 
Implemented the review comment.
Please Review

**Comment by btshimizukza** [comment]
@btundes 

A modification is good!
But please change a wording into commit log of git.
(A link to the issue page is not added)

Before : `Issue#1110`
After : `#1110`

Procedures are follows;

1. reset a commit

 ```
 git reset --soft 7adfc2ea1aa6ad67cf9c2c895e955b39ef907deb
 ```

2. commit again with new commit comment

 ```
 git commit -m ".... #1110"
 ```

3. push a changes

 ```
 git push -f origin issues/1110_Correcting_Authentication_Request_Matcher
 ```

**Comment by btshimizukza** [comment]
@btundes 

Please backport to 5.0.x.

**Pull request merged**